packages/preact/test/index.test.tsx

```diff
@@ -239,60 +240,56 @@ describe("@preact/signals", () => {
 			expect(text.textContent).to.equal("d");
 		});
 
-		describe("garbage collection", function () {
-			// Skip GC tests if window.gc/global.gc is not defined.
-			before(function () {
-				if (typeof gc === "undefined") {
-					this.skip();
-				}
-			});
+		describe.skipIf(typeof gc === "undefined")(
```

Comment:
the formatter did most of this diff 😬  thanks to `skipIf` making the line longer